moderation_queue/forms.py

```diff
@@ -33,11 +53,27 @@ def clean(self):
         why_allowed = cleaned_data.get('why_allowed')
         if why_allowed == 'other' and not justification_for_use:
             message = _("If you checked 'Other' then you must provide a "
-                "justification for why we can use it.")
+                        "justification for why we can use it.")
             raise ValidationError(message)
         return cleaned_data
 
 
+class UploadPersonPhotoUrlForm(forms.Form):
+    image_url = StrippedCharField(
+        widget=forms.URLInput(),
+    )
+    why_allowed_url = forms.ChoiceField(
```

Comment:
Mmm... but the HTML will be generated with an `id` attribute that is called `why_allowed` non?
Which means we will have two elements in the same page with the same `id` attribute :thinking: